sql/popularize.go

```diff
@@ -0,0 +1,36 @@
+package sql
+
+// Copyright 2019 The SQLFlow Authors. All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import (
+	"strings"
+)
+
+// Popularize reads SQL statements from the file named *.sql
+// and runs each SQL statement with db.
+func Popularize(db *DB, sqlcontent string) error {
```

Comment:
Put `Popularize` in package `sql` but not in `testdata` so that would not generate cycle import

Reply:
You can use `database/sql` to avoid circular import. In `package testdata`
```go
package testdata

import (
	"strings"
	"database/sql"    // not github.com/sql-machine-learning/sqlflow/sql
)

func Popularize(db *sql.DB, sqlcontent string) error { ... }
```

Then in other packages,

```go
package main

import (
	"github.com/sql-machine-learning/sqlflow/sql"
	"github.com/sql-machine-learning/sqlflow/sql/testdata"
)

func main() {
	sqlflowDB := sql.Open(...)
	testdata.Popularize(sqlflowDB.DB, testdata.IrisSQL) // notice sqlflowDB.DB
}
```

Reply:
Done.